### PR TITLE
Script to deploy linux wheel to S3 on release.

### LINKED ISSUE #1744: Linux wheels deploy to S3 on release
We can't upload the wheel to PyPi because PyPi rejects linux platform wheel files. So we'll push it up into S3. See: https://bitbucket.org/pypa/pypi-metadata-formats/issue/15/enhance-the-platform-tag-definition-for

### PR DESCRIPTION
**Pull request by rhyolight**
Fixes #1744.

### CODE REVIEW

**Comment by rhyolight** [comment]
@oxtopus Please review. I've tested the deploy script locally, but will have to push a release to test in Travis-CI.

File: ci/travis/after_success-release-linux.sh [hunk 1]
```diff
@@ -32,11 +34,12 @@ sudo pip install twine || exit
 echo "Creating distribution files..."
 # This release build creates the source distribution. All other release builds
 # should not.
-python setup.py sdist bdist || exit
+python setup.py sdist bdist bdist_wheel || exit
```
Comment (oxtopus) [inline ci/travis/after_success-release-linux.sh]: Why use twine below in lieu of `python setup.py sdist bdist bdist_wheel upload` ?

Comment (rhyolight) [inline ci/travis/after_success-release-linux.sh]: Following the recommendation of the official [Python Packaging Guide](https://packaging.python.org/en/latest/distributing.html#uploading-your-project-to-pypi).

> (Recommended): Use twine
>
>     twine upload dist/*
> The biggest reason to use twine is that python setup.py upload (option #2 below) uploads files over plaintext. This means anytime you use it you expose your username and password to a MITM attack. Twine uses only verified TLS to upload to PyPI protecting your credentials from theft.

**Comment by oxtopus** [comment]
:+1:

**Pull request merged**